lte/gateway/c/core/oai/tasks/sgw/spgw_state_converter.cpp

```diff
@@ -215,8 +215,10 @@ void SpgwStateConverter::sgw_pdn_connection_to_proto(
 
 void SpgwStateConverter::proto_to_sgw_pdn_connection(
     const oai::SgwPdnConnection& proto, sgw_pdn_connection_t* state_pdn) {
-  state_pdn->apn_in_use =
-      strndup(proto.apn_in_use().c_str(), proto.apn_in_use().length());
+  if ((proto.apn_in_use().length())) {
```

Comment:
Can't we use  something like `proto.has_apn_in_use()` here ? Ref: https://developers.google.com/protocol-buffers/docs/reference/cpp-generated#embeddedmessage

Reply:
Got it, thanks!